example_scripts/WCSSP_India_example_plot.py

```diff
@@ -22,10 +22,10 @@
 dir_nam = "/projectsa/COAsT/NEMO_example_data/MO_INDIA/"
 fil_nam = "ind_1d_cat_20180101_20180105_25hourm_grid_T.nc"
 dom_nam = "domain_cfg_wcssp.nc"
+config_t = "/work/jelt/GitHub/COAsT/example_files/example_t_nemo_config.json"
```

Comment:
Before this gets merged, can we change this to something generic (i.e. using "~") and platform-independent (i.e. using Path)?